docs/source/keywords.rst

```diff
@@ -19,76 +19,86 @@ This page is organized in four sections:
 
 .. _keywords_backends:
 
-Kokkos backends
-===============
+Backend selection
+=================
+
+**Default State:**
+All backends are disabled by default.  This ensures you explicitly choose the
+backends you need for your specific hardware setup.
+
+**Enabling Backends:**
+You can enable backends by configuring with ``-DKokkos_ENABLE_<BACKEND>=ON``
+flag, where ``<BACKEND>`` is replaced with the specific backend you want to
+enable (e.g. ``-DKokkos_ENABLE_CUDA=ON`` for CUDA).
+
+**Restrictions:**
+  Mutual Exclusion: You can only have one device backend (e.g., CUDA, HIP,
+  SYCL) and one host parallel backend (e.g., OpenMP, C++ threads) enabled at
+  the same time. This is because these backends manage parallelism in
+  potentially conflicting ways.
```

Comment:
```suggestion
  the same time. This is because these backends manage parallelism in
  potentially conflicting ways.
```
Does this argument hold for the device backends?

Reply:
Your suggestion is invalid (the code didn't change)

> Does this argument hold for the device backends?

Yes I think so